swagger.yml

```diff
@@ -2257,6 +2257,8 @@ definitions:
         type: boolean
       functions:
         type: object
+      branch:
```

Comment:
Is this also part of the HTTP response (which is the `deploy` definition I believe)?

Reply:
No, this only exists for `createSiteDeploy` endpoint